config/config.go

```diff
@@ -72,6 +72,25 @@ func (g Generator) OutPath(packageDir string) string {
 	return filepath.Join(g.ConfigDir, g.Out)
 }
 
+// addDefaults will add any required default values to
+// the generator.
+func (g *Generator) addDefaults() {
+	switch true {
```

Comment:
remove `true`? I think `switch true` and `switch` are both the same thing?

Reply:
Ah, thanks!